cli/tests/integrations/test_package.py

```diff
@@ -207,7 +207,8 @@ def test_describe():
     "framework"
   ],
   "versions": [
-    "0.8.1"
+    "0.8.1",
+    "0.9.0-RC3"
```

Comment:
@BenWhitehead @mgummelt Do we know why the Marathon package has multiple versions? Is this the intended usage?